Source/Core/Core/IOS/ES/TitleManagement.cpp

```diff
@@ -536,24 +526,25 @@ ReturnCode ES::DeleteTicket(const u8* ticket_view)
   ticket.DeleteTicket(ticket_id);
 
   const std::vector<u8>& new_ticket = ticket.GetBytes();
-  const std::string ticket_path = Common::GetTicketFileName(title_id, Common::FROM_SESSION_ROOT);
+  const std::string ticket_path = Common::GetTicketFileName(title_id);
+  if (!new_ticket.empty())
   {
-    File::IOFile ticket_file(ticket_path, "wb");
-    if (!ticket_file || !ticket_file.WriteBytes(new_ticket.data(), new_ticket.size()))
+    const auto file = m_ios.GetFS()->OpenFile(0, 0, ticket_path, FS::Mode::ReadWrite);
```

Comment:
This might be another candidate to cache the FS result.

Reply:
I agree, I've done it anywhere else where it's used more than a couple of times.